src/UI/examples/ViewControl/Section/base.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+function base() {
+	//Loading factories
+	global $DIC;
+	$f = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	$back = $f->button()->standard("a", "http://www.ilias.de");
+	$next = $f->button()->standard("b", "http://www.github.com");
```

Comment:
Chose next as label in the example here. Previous is a better choice, since it tells something about the role of the button.